### PR TITLE
[Fabric-Sync] Simplify and update the fabric sync guide

### PR DESCRIPTION
**Pull request by yufengwangca**
1. Simplify the setup scripts and update the fabric sync guide
2. Disable the auto-sync feature to prevent cyclic syncing

### CODE REVIEW

File: docs/guides/fabric_synchronization_guide.md [hunk 1]
```diff
@@ -114,36 +130,54 @@ Pair the Light Example with node ID 3 using its payload number:
...
-After the Light Example is successfully paired in Fabric-Source, it will be
-synced to Fabric-Sink with a new assigned node ID.
+After the device is successfully added, you will observe the following on
+Ecosystem 2 with the newly assigned Node ID:
+
+```
+>>> New device with Node ID: 0x3 has been successfully added.
+```
+
+Additionally, you should also get notified an new device is added on the
```
Comment (saurabhst) [inline docs/guides/fabric_synchronization_guide.md:140]:
```suggestion
Additionally, you should also get notified when a new device is added to
```

File: examples/fabric-admin/commands/fabric-sync/Commands.h [hunk 2]
```diff
@@ -29,7 +29,6 @@ void registerCommandsFabricSync(Commands & commands, CredentialIssuerCommands *
         make_unique<FabricSyncAddBridgeCommand>(credsIssuerConfig),
         make_unique<FabricSyncRemoveBridgeCommand>(credsIssuerConfig),
         make_unique<FabricSyncDeviceCommand>(credsIssuerConfig),
-        make_unique<FabricAutoSyncCommand>(credsIssuerConfig),
```
Comment (tehampson) [inline examples/fabric-admin/commands/fabric-sync/Commands.h:32]: nit, why is this getting removed here?

If you do want to remove this why not also removed from `examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h`

Comment (saurabhst) [reply examples/fabric-admin/commands/fabric-sync/Commands.h:32]: > nit, why is this getting removed here?
> 
> If you do want to remove this why not also removed from `examples/fabric-admin/commands/fabric-sync/FabricSyncCommand.h`

+1 I have the same concern/comment/clarification

File: docs/guides/fabric_synchronization_guide.md [hunk 3]
```diff
@@ -114,36 +130,54 @@ Pair the Light Example with node ID 3 using its payload number:
...
+Ecosystem 2 from the Ecosystem 1:
+
+```
+>>> A new devie is added on Endpoint 3.
```
Comment (tehampson) [inline docs/guides/fabric_synchronization_guide.md:144]: nit
```suggestion
>>> A new device is added on Endpoint 3.
```

File: examples/fabric-admin/commands/fabric-sync/Commands.h (hunk 2 above)
Comment (yufengwangca) [reply examples/fabric-admin/commands/fabric-sync/Commands.h:32]: Auto-sync will cause cyclical syncing at this moment,  but we should be able to solve this with proper algorithm using uniqueID.

I don't see this feature in the test plan, do we want to remove this feature for good? then I will clean up all related pieces of this feature.

Comment (tehampson) [reply examples/fabric-admin/commands/fabric-sync/Commands.h:32]: We can leave the command in and mark the command as use at your own risk, but I don't understand why we are removing it

Comment (yufengwangca) [reply examples/fabric-admin/commands/fabric-sync/Commands.h]: Acked

**Review by saurabhst — APPROVED** [review]

**Review by tehampson — APPROVED** [review]

**Pull request merged**